### DISCUSSION TITLE
MvxSplashScreenActivity changed

### DISCUSSION THREAD

**Discussion by JimWilcox3** [MvvmCross/MvvmCross#4679] [General]
After looking through the playground, I discovered that it is now called MvxStartActivity.  The readme that pops up when you install the MvvmCross Nuget still directs you to create an MvxSplashScreenActivity.  The sample template also has this.  It was a little confusing to troubleshoot.  After switching to MvxStartActivity, my app won't run.  I tracked the error down to the bundle being null in OnCreate for MvxStartActivity.  I think it has to do with the device I am using runs Android 30 and the call to base my splash screen shows it isn't available until a later version.

Jim

**Comment by Cheesebaron** [comment]
I am happy to hop on a call with you to fix your issue. You can jump in the #mvvmcross channel on Discord and ping me: https://aka.ms/dotnet-discord

**Reply by Cheesebaron** [reply]
But I do agree, we need to improve the getting started docs and templates when we do bigger breaking changes like this. Sorry for the confusion.
